validator/rpc/server.go

```diff
@@ -187,6 +187,7 @@ func (s *Server) Start() {
 	pb.RegisterHealthServer(s.grpcServer, s)
 	pb.RegisterBeaconServer(s.grpcServer, s)
 	pb.RegisterAccountsServer(s.grpcServer, s)
+	pb.RegisterSlashingProtectionServer(s.grpcServer, s)
```

Comment:
You will also need to register this in validator/rpc/gateway/gateway.go as pb.RegisterSlashingProtectionHandlerFromEndpoint

Reply:
Done